docs/source/notes/distributed_autograd.rst

```diff
@@ -367,15 +367,9 @@ autograd and distributed optimizer is as follows:
       rpc.wait_all_workers()
 
   file_name = NamedTemporaryFile().name
-  processes = []
 
   # Run two workers.
-  for i in range(2):
-      p = mp.Process(target=run_process, args=(i, (i + 1) % 2, file_name))
-      p.start()
-      processes.append(p)
-
-  for p in processes:
-      p.join()
+  nprocs = 2
+  mp.spawn(_run_process, args=(i, (i + 1) % 2, file_name), nprocs)
```

Comment:
It seems we can drop the first arg `i` in the `args` tuple? Otherwise, the actual args passed to `fn` would be `i, i (i 
+ 1) % 2`?

> The function is called as `fn(i, *args)`, where i is the process index and args is the passed through tuple of arguments.

Reply:
Also, specify `nprocs` as kwarg (after another kwarg).